### ISSUE TITLE
MiST: scandoubler_disabled and csync_disable

### ISSUE THREAD

**Issue by jotego** [jotego/jtframe#50]
From Phil (Patreon):

```
I noticed that scandoubler_disabled = 1 (15khz) 
and csync_disable = 1 (separate horizontal and vertical sync) 
do not work together in many of your arcade cores, like CPS2. 

This functionality was introduced with firmware_200208, 
but I guess it needs to be added to cores specifically.
```

**Comment by gyurco** [comment]
no_csync is just a bit returned in user_io, it's up to the core to use it.
Example how the default mist_video module handles this flag:
https://github.com/mist-devel/mist-modules/blob/master/mist_video.v#L153

**Comment by jotego** [comment]
Thank you. I have fixed it in b95aa78. It will be part of updated compilations.
